docs/source/user_guide/pipeline-components.md

```diff
@@ -50,18 +50,186 @@ Component details and demo pipelines can be found in the `https://github.com/ely
+
+##### Name (display_name)
+
+A user-friendly name for the registry entry. Note that the registry entry name is not displayed in the palette. This property is required.
+
+Example: `data load components`
+
+##### N/A (name)
+
+A unique identifier for this registry entry. A value is generated from `Name` if no value is provided.
+
+Example: `data_load_components`
+
+##### Description (description)
+
+A description for the registry entry.
+
+Example: `Load data from external data sources`
+
+##### Category (categories)
+
+In the pipeline editor palette components are grouped into categories to make them more easily accessible. If no category is provided, the components defined by this registry entry are added to the palette under `no category`.  
+
+Examples (CLI):
+
+- `['load data from database']`
+- `['train model','pytorch']`
+
+
+##### Runtime (runtime)
+
+The runtime environment that supports the component(s). Valid values are `kfp` and `airflow`. This property is required.
+
+Example:
+
+- `airflow`
+
+##### Location Type (location_type)
+
+The location type identifies the format that the value(s) provided in `Paths` represent. Supported types are `URL`, `Filename`, or `Directory`. This property is required.
+
+- `URL`: The provided `Paths` identify web resources. The pipeline editor loads the specified URLs using anonymous HTTP `GET` requests.
+- `Filename`: The provided `Paths` identify files in the file system where JupyterLab/Elyra is running. `~` may be used to denote the user's home directory.
+
+The `Directory` location type is used to _add components during the Elyra build process_. The provided `Paths` must identify existing directories in [`etc/config/components`](https://github.com/elyra-ai/elyra/tree/master/etc/config/components).
+
+
+##### Paths (paths)
+
+A path defines the location from where the pipeline editor loads one or more component specifications. The provided value must be a valid representation of the selected _location type_. This property is required.
+
+Examples (GUI):
+ - URL: `https://raw.githubusercontent.com/elyra-ai/elyra/master/etc/config/components/kfp/run_notebook_using_papermill.yaml`
+ - Filename: `/Users/patti/specs/load_data_from_public_source/http_operator.py`
+ - Filename: `~patti/specs/filter_files/row_filter.yaml`
+ - Directory path: `load_from_database`
+
+ Examples (CLI):
+ - URL: `['https://raw.githubusercontent.com/elyra-ai/elyra/master/etc/config/components/kfp/run_notebook_using_papermill.yaml']`
+ - Filename: `['/Users/patti/specs/load_data_from_public_source/http_operator.py']`
+ - Filename: `['~patti/specs/filter_files/row_filter.yaml']`
+ - Directory path: `['load_from_database']`
+
+ Examples multiple components (CLI):
+ - URL: `['URL1', 'URL2']`
+ - Filename: `['/Users/patti/specs/comp1.yaml','/Users/patti/specs/comp2.yaml']`
```

Comment:
Seems like a multi-valued example for `Directory` is warranted...
```
- Directory: `['/Users/patti/specs','/Users/patti/specs2']`
```

Reply:
added!